File: server/routes/item_routes.js
const express = require('express');
const { getItems, addItem } = require('../controllers/item_controllers');
// const {ensureAuth, ensureGuest} = require("../middleware/auth");
// const extractAdminFromCookie = require('../middleware/adminAuth');

const router = express.Router();


router.route('/get-items').get(getItems);
router.route('/add-item').post(
    // extractAdminFromCookie,
     addItem);


module.exports = router;

File: server/go/routes/item_routes.go
package routes

import (
	"igrosine/controllers"
	middlewars "igrosine/middleware"

	"github.com/gofiber/fiber/v2"
)

func ItemRoutes(app *fiber.App) {
	app.Get("/api/get-items", controllers.GetItems)
	app.Post("/api/add-item",
	middlewars.Auth, 
	controllers.AddItem)
	app.Post("/api/delete-item",
	
	middlewars.Auth,
	
	controllers.DeleteItem)
}
